docs/assets/bindings-get-flow.drawio

```diff
@@ -0,0 +1,100 @@
+<mxfile host="Electron" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/24.7.17 Chrome/128.0.6613.36 Electron/32.0.1 Safari/537.36" version="24.7.17">
```

Comment:
In `create` we end diagram with `Return DB Binding` but in `get` with `Return Binding`